src/seedu/addressbook/common/Utils.java

```diff
@@ -34,4 +36,15 @@ public static boolean elementsAreUnique(Collection<?> items) {
         }
         return true;
     }
+
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public static String getPrintableString(Printable... printables) {
```

Comment:
Great use of varargs to allow multiple printables to be passed into the method =)!.